src/index.html

```diff
@@ -8,12 +8,12 @@
 </head>
 <body>
    <header>
-        <h1>Leaderboard</h1>
+        <h1>⭐️🌸😊 Welcome to my Leaderboard 😊🌸⭐️</h1>
    </header>
    <section class="main">
         <article class="score">
             <div class="info">
-                <h2 class="title">Recent scores</h2>
+                <h2 class="title">Recent scores 🥇🎖🏆</h2>
                 <button type="button" class="btn-refresh">Refresh</button>
```

Comment:
> - [ ] Impressive job on the aesthetically pleasing app styling!👍 However, I've noticed that the refresh button is not aligned with the recent score text and also the "submit" `button` is not aligned properly with the provided design. I suggest adjusting the styling of the recent scores and the submit button to match the provided wireframe design. This attention to detail will enhance the overall visual consistency.😊
> 
> Provided design wireframe:
> 
> ![red](https://github.com/IvonneBenitesRodriguez/leaderboard-microverse/assets/40009354/c5e5b0ad-59d9-4f79-b2ff-ea94f045ffd6)
> 
> Your design:
> 
> ![red1](https://github.com/IvonneBenitesRodriguez/leaderboard-microverse/assets/40009354/0844c58e-886c-4b37-9a9e-b4cec33402ae)
>

Reply:
Hello @Johnadibe 👋🏾
Very nice to meet you
Thank you for the suggestions, I appreciate it,
I modified it according to what you suggested,
thank you again,
Best wishes,
Ivonne